gitlab/v4/cli.py

```diff
@@ -280,6 +284,11 @@ def _populate_sub_parser_by_class(
                         f"--{x.replace('_', '-')}", required=False
                     )
 
+            if mgr_cls._create_attrs.exclusive:
+                group = sub_parser_action.add_mutually_exclusive_group()
+                for x in mgr_cls._create_attrs.exclusive:
```

Comment:
Should this be '_update_attrs`?

Also could the commits be squashed down to one commit?

Reply:
Indeed. This was a copy/paste error. I bit the bullet and added a unit test to prevent mistakes like this.